docs/pages/choose-an-edition/teleport-enterprise/introduction.mdx

```diff
@@ -80,7 +80,9 @@ See our [FedRAMP Compliance for Infrastructure Access](../../access-controls/com
 
 ## Access Requests
 
-With Teleport we've introduced the ability for users to request additional roles. The Access Request API makes it easy to dynamically approve or deny these requests.
+With Teleport users can request access to specific resources and roles with requests publishable to tools such as Slack, Jira, PagerDuty that fits to your organization's workflows. Authorized users and automated processes can approve the requests. The commercial edition of Access Requests allows for
+requesting specific resources such as a production server to allow fine-grain access for a targeted task.  The Web User Interface for Access Requests
```

Comment:
The web interface for creating and approving access requests is only available in the Teleport Enterprise.